docs/user/contextplugin.md

```diff
@@ -0,0 +1,27 @@
+# FlowForge Persistent Context Plugin
+
+Some Node-RED flows need a way to persist context values between restarts and FlowForge stack 
+updates but context data is ephemeral by default. With FlowForge Premium, persistent context 
+storage is possible and it survives restarts, upgrades and more.
+
+## Usage
+
+In your Node-RED, you will now have the option to store values in 2 context stores...
```

Comment:
Why 3 dots?

```suggestion
In your Node-RED, you will now have the option to store values in 2 context stores.
```

Also; why is this an option? It goes against: https://flowforge.com/handbook/product/plan/#convention-over-configuration

Reply:
I might be using the elipsis incorrectly. Will change to a colon to suggest the next line(s) are related

> Also; why is this an option? It goes against: https://flowforge.com/handbook/product/plan/#convention-over-configuration

That document is not relevant to Node-RED context. Users decide to use volatile or persistent context based on the flow requirements. Removing choice here would be harmful